app/routines/update_books_from_salesforce.rb

```diff
@@ -0,0 +1,27 @@
+class UpdateBooksFromSalesforce
+
+  def self.call
+    new.call
+  end
+
+  def log(message, level = :info)
+    Rails.logger.tagged(self.class.name) { Rails.logger.public_send level, message }
+  end
+
+  def call
+    log('Starting UpdateBookSalesforceInfo')
+
+    books_updated = 0
+    sf_books      = OpenStax::Salesforce::Remote::Book.order(:Id)
+
+    sf_books.each do |sf_book|
+      book                 = Book.find_or_initialize_by(salesforce_id: sf_book.id)
+      book.salesforce_name = sf_book.name
+      book.official_name   = sf_book.official_name
+
+      books_updated        += 1 if book.save!
```

Comment:
`book.save!` just throws on failure
if you want a boolean, you want `book.save` instead

Reply:
Makes sense - thanks! Fixed.